README.md

```diff
@@ -190,6 +190,18 @@ toLowerCase({ nested: { prop: { 1: { msg: 'Hello' }, 2: { msg: 'Hi' }, 3: { msg
 
 When applying operations on a path immutad●t tries to create the minimum of objects or arrays needed to guarantee your data structure to be immutable.
```

Comment:
Put that under benchmark, this is a memory related information, memory is less relevant than speed to JS developers...